packages/core/contracts/insured-bridge/non-bridge-helpers/BridgePoolSynchronizer.sol

```diff
@@ -0,0 +1,32 @@
+pragma solidity ^0.8.0;
+
+interface BridgePoolLike {
+    function syncUmaEcosystemParams() external;
+
+    function syncWithBridgeAdminParams() external;
+}
+
+/**
+ * @notice Small helper contract to make calling sync methods on a set of bridge pools easier. When one pool needs to be
+ * sync it is likely that all ecosystem pools also need to be synced. This contract lets the caller do this in a batch.
+ */
+contract BridgePoolSynchronizer {
+    function syncUmaEcosystemParams(BridgePoolLike[] memory bridgePools) public {
+        for (uint256 i = 0; i < bridgePools.length; i++) {
+            bridgePools[i].syncUmaEcosystemParams();
```

Comment:
nitty nit: what if some bridge pools we want to skip for some reason and get some gas-savings? Is that possible?

Reply:
just dont put them in the array of `bridgePools` that you feed in

Reply:
true!